scripts/staff/staff_roll_gui.py

```
@@ -0,0 +1,723 @@
+
+        # 处理左侧行
+        if cmd.row_left != -1:
+            hpt_idx, div_idx = self.assets._row_map.get(cmd.row_left, (None, None))
+            log_msg += f"Left: RowID {cmd.row_left} (HPT {hpt_idx:02d}, Div {div_idx}) | "
+            
+            img_l, w_l, h_l = self.assets.photo(cmd.row_left, 1.0, self.root)
+            if img_l is not None and w_l > 0:
+                w_left = w_l
+                a = cmd.align
+                # 坐标计算逻辑保持不变...
+                if a == 0: x = 240 - w_l
```

Ruff (0.15.17)

[error] Multiple statements on one line (colon)

(E701)